### PR TITLE
Default VS Code builds to net5.0

### PR DESCRIPTION
**Pull request by deanmarcussen**

### CODE REVIEW

**Comment by Skrypt** [comment]
I'm guessing that we should now have one of these tasks/launcher per framework target.

File: .vscode/launch.json [hunk 1]
```diff
@@ -2,10 +2,10 @@
 	"version": "0.2.0",
 	"configurations": [
 		{
-			"name": ".NET Core Launch (console)",
+			"name": ".NET Core Launch (console net5.0 debug)",
```
[the remark this answers is not in the document]
Comment (Skrypt) [reply .vscode/launch.json:5]: I'd prefer using a suffix like this

**Comment by deanmarcussen** [comment]
Yeah sound fine to me. I might push though, because it's reformatted the document weirdly.

**Comment by Skrypt** [comment]
I format it inside VS Code that's why.

**Comment by deanmarcussen** [comment]
haha so did I ;)

**Comment by Skrypt** [comment]
Yeah but now the indentation is different from one file to the other.

**Comment by deanmarcussen** [comment]
Change it back if you wish. It was just completely off before, (tabs vs spaces)

**Comment by Skrypt** [comment]
Yeah, I don't care much about the diff here if the document is not formatted properly from the beginning. Next time the diff will be working.

**Review by Skrypt — APPROVED** [review]

**Pull request merged**